proto/zigbeealliance/distributedcomplianceledger/model/tx.proto

```diff
@@ -42,11 +42,11 @@ message MsgCreateModel {
   string lsfUrl = 17 [(gogoproto.moretags) = "validate:\"omitempty,url,startsnotwith=http:,max=256\""];
   uint32 schemaVersion = 18 [(gogoproto.moretags) = "validate:\"gte=0,lte=65535\""];
   int32 enhancedSetupFlowOptions = 19 [(gogoproto.moretags) = "validate:\"omitempty,gte=0,lte=65535\""];
```

Comment:
Do we need to fix startsnotwith=http: for other fields?

Reply:
Yep. Fixed

Reply:
Had to revert it, as actually it says "All HTTP based URLs SHALL use the https scheme", not that it shall be HTTPS only.